x-pack/solutions/security/packages/kbn-securitysolution-utils/src/time_duration/to_largest_time_duration.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Converts milliseconds to time duration
+ */
+export function toLargestTimeDuration(ms: number): string {
```

Comment:
Shouldn't it return `TimeDuration`?

```suggestion
export function toLargestTimeDuration(ms: number): TimeDuration {
```

If we make a bunch of utilities around time duration, there should be a single way in (parsing), one or many ways out (conversion to other types, such as a number of milliseconds or a string representation), and the rest would be calculation utilities that work with instances of `TimeDuration` only. This is how `moment` and other libraries work, and this allows for SOLID and DRY.